src/AppInstallerCommonCore/Public/AppInstallerDownloader.h

```diff
@@ -31,8 +43,10 @@ namespace AppInstaller::Utility
     std::optional<std::vector<BYTE>> Download(
         const std::string& url,
         const std::filesystem::path& dest,
+        DownloadType type,
         IProgressCallback& progress,
-        bool computeHash = false);
+        bool computeHash = false,
+        std::string_view downloadIdentifier = {});

```

Comment:
>downloadIdentifier [](http://example.com/codeflow?start=25&length=18)

Should we add this to the above comments? Something like, only used in DO as content id